composer.json

```diff
@@ -13,29 +13,35 @@
             "name": "Casper Bottelet",
             "email": "cbottelet@gmail.com",
             "role": "Developer"
+        },
+        {
+            "name": "Jason Huebel",
+            "email": "jason@huebel.dev",
+            "role": "Contributor"
         }
     ],
     "license": "MIT",
     "type": "project",
     "require": {
-        "php": ">=5.6.4",
-        "laravel/framework": "5.4.*",
-        "laravelcollective/html": "5.4.*@dev",
-        "yajra/laravel-datatables-oracle": "~6.0",
-        "guzzlehttp/guzzle": "^6.2",
-        "pusher/pusher-php-server": "^2.3",
-        "zizaco/entrust": "1.7.*",
+        "php": ">=7.1.3",
+        "laravel/framework": "5.8.*",
```

Comment:
I'm pretty sure this requires a lot more work then just this. Have you followed the documentation?

Reply:
I followed the upgrade instructions for each point update from 5.4 through 5.8. Since there isn't any core class overriding, most of the fixes were pretty simple as I progressed through the updates.